starboard/tools/testing/test_runner.py

```diff
@@ -477,7 +477,7 @@ def MakeLauncher():
       logging.info(("Xml results for this test will "
                     "be logged to '%s'."), test_result_xml_path)
     elif self.xml_output_dir:
-      xml_output_subdir = os.path.join(self.xml_output_dir, target_name)
```

Comment:
I have not found any usage of this flag. It was added in b/112477847 that looks like an experiment to run cobalt builds on an internal build system. I think this is safe to change.

Reply:
This should be safe to remove yep.